app/views/index.html

```diff
@@ -46,6 +46,7 @@
             window.__ROOM_NAME__ = "${roomName}";
             window.__USER_ADMIN__ = "${user.isAdmin()}";
             window.__USER_GUEST__ = "${user.isGuest()}";
+            document.title="${room_name"} + " : breaker";
```

Comment:
No, I always see just "breaker" , no matter what the room I am in.

On 29-Aug-16 15:55:45, "Rick Hanlon II" <notifications@github.com> 
wrote:

>In app/views/index.html:
>
> > @@ -46,6 +46,7 @@ > window.__ROOM_NAME__ = "${roomName}"; > 
>window.__USER_ADMIN__ = "${user.isAdmin()}"; > window.__USER_GUEST__ = 
>"${user.isGuest()}"; > + document.title="${room_name"} + " : breaker";
>n.b. the document title should be getting set via this component. Were 
>you not seeing it set?
>
>—
>You are receiving this because you authored the thread.
>Reply to this email directly, view it on GitHub, or mute the thread.
>